java/sage/SchedulerSelector.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2015 The SageTV Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sage;
+
+public class SchedulerSelector
+{
+  private static final SchedulerInterface ourInstance;
+
+  public static SchedulerInterface getInstance()
+  {
+    return ourInstance;
+  }
+
+  static
+  {
+    // TODO: This will be pulled together in a later commit.
+    if (SeekerSelector.USE_BETA_SEEKER)
+      ourInstance = null; //Scheduler2.getInstance();
+    else
+      ourInstance = Scheduler.getInstance();
```

Comment:
The idea is that it only loads one or the other. If we put that logic in getInstance(), we are branching for every call to Seeker which is what we are trying to avoid. We will never be changing between the two implementations at runtime since the results would be unpredictable. Would it be acceptable to just have it in a SchedulerInterfaceHolder class?